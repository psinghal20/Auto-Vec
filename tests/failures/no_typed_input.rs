use auto_vec::auto_vec;

struct T;

impl T {
    #[auto_vec]
    fn foo(&self) -> usize {
        return 5;
    }
}

fn main() {}