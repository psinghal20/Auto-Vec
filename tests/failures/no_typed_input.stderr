error: Auto_vec: Expected one or more Typed arguments, Found None
 --> $DIR/no_typed_input.rs:7:5
  |
7 |     fn foo(&self) -> usize {
  |     ^^^^^^^^^^^^^^^^^^^^^^
